gremlin-python/src/main/python/gremlin_python/driver/request.py

```diff
@@ -20,6 +20,5 @@
 
 __author__ = 'David M. Brown (davebshow@gmail.com)'
 
-
-RequestMessage = collections.namedtuple(
-    'RequestMessage', ['processor', 'op', 'args'])
+RequestMessageV4 = collections.namedtuple(
+    'RequestMessageV4', ['fields', 'gremlin'])
```

Comment:
FYI, you don't necessarily need to make the RequestMessageV4 like this. In Java, 'gremlin' was split away from 'fields' because 'gremlin' is the only field that will always exist, whereas the other items in 'fields' might change. This is to ensure that the request will always be compatible. You could just put 'gremlin' as just another key inside the map if it is easy for you do so in Python.

Reply:
It's actually ok, but I'll keep this in mind for when we need to do refactoring.